### PR TITLE
Update labels.yaml with sig labels

### PR DESCRIPTION
**Pull request by spiffxp**
**What this PR does / why we need it**: Updates labels.yaml to reflect the state of labels in this repo.  Consumed by https://github.com/kubernetes/test-infra/blob/master/mungegithub/mungers/check-labels.go to ensure these labels continue to exist.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by spiffxp** [comment]
/cc @kubernetes/sig-testing-pr-reviews @kubernetes/sig-contributor-experience-pr-reviews 

(unclear who owns the munger)

**Comment by spiffxp** [comment]
FWIW there's a discrepancy between the label names and the folder names in kubernetes/community.  It might be useful for tooling purposes to unify them.
```
aaron@cloudtop:kubernetes (update-labels-yaml %)$ grep sig/ labels.yaml | awk '{print $3}' > ~/Desktop/sig-labels.txt
aaron@cloudtop:community (master)$ find . -type d -name 'sig-*' -maxdepth 1 | tr -d './' | sed -e 's|sig-|sig/|' > ~/Desktop/sig-folders.txt
aaron@cloudtop:community (master)$ diff ~/Desktop/sig-{labels,folders}.txt
10c10
< sig/contributor-experience
---
> sig/contribx
16c16
< sig/onprem
---
> sig/on-prem
17a18
> sig/pm
23a25
> sig/ui
```

File: labels.yaml [hunk 1]
```diff
@@ -1,285 +1,336 @@
...
+  color: d2b48c
+- name: team/huawei
+  color: d2b48c
+- name: team/mesosphere
+  color: d2b48c
+- name: team/redhat
+  color: d2b48c
+- name: team/test-infra
+  color: ededed
+- name: team/ux (deprecated - do not use)
```
Comment (xiangpengzhao) [inline labels.yaml]: Just a question: why don't we remove these deprecated labels? I guess they are for the legacy issues and PRs which already have had such labels.

**Comment by thockin** [comment]
/approve

I'll approve - but who should review?

File: labels.yaml (hunk 1 above)
Comment (spiffxp) [inline labels.yaml]: I'd like to see them gone too, but wanted to keep this PR trivial.  I defer to @kubernetes/sig-contributor-experience-misc for modification of labels since I don't have write privs here.  Unclear to me whether removing, renaming, or swapping labels would alter github issue recency.

Comment (xiangpengzhao) [inline labels.yaml]: yeah, removing these labels might cause problem to issues with such labels. It'd be better to replace them with alternative labels for those issues automatically. @fejta is there any way doing this?
TBH, I don't like deprecated things, it always make me confused: why are they deprecated? when will they be removed? what's the alternative? etc.

Comment (spiffxp) [inline labels.yaml]: `labeler` can rename labels, and I'm sure prow or mungegithub could help here, just unsure whether it'd modify recency (or if we care)

**Comment by spiffxp** [comment]
>I'll approve - but who should review?

ah the joys of touching a file in the root of the project; would anyone from @kubernetes/sig-contributor-experience-pr-reviews like to chime in?

**Comment by fejta** [comment]
How did you generate this? Ideally there's a command we can run

**Comment by spiffxp** [comment]
I used https://github.com/tonglil/labeler
```
cd $GOPATH/src/k8s.io/kubernetes
labeler scan labels.yaml --repo kubernetes/kubernetes
```

File: labels.yaml (hunk 1 above)
Comment (fejta) [inline labels.yaml]: What are the alternatives?

I agree with Aaron that figuring out how to retire labels seems outside the scope of this PR...

**Comment by fejta** [comment]
I guess I should learn to read comments...
